packages/iotex-react-block-producers/src/partition-help.ts

```diff
@@ -55,3 +55,7 @@ export function getClassifyDelegate(
     ...candidates
   ];
 }
+
+export function isProbation(record: CustomTBpCandidate): boolean {
```

Comment:
you don't need this function, just one line of 

`Boolean(record.probation)` will do the work